Sources/BraveWallet/Crypto/Stores/SelectAccountTokenStore.swift

```diff
@@ -114,7 +120,8 @@ class SelectAccountTokenStore: ObservableObject {
     walletService: BraveWalletBraveWalletService,
     assetRatioService: BraveWalletAssetRatioService,
     ipfsApi: IpfsAPI,
-    userAssetManager: WalletUserAssetManagerType
+    userAssetManager: WalletUserAssetManagerType,
+    query: String? = nil
```

Comment:
Might be a better idea to use 
`query: String = ""
`
and use 
`self.query = query`

Reply:
Yeah I have no strong preference here, I'll update to sync with `resetFilters()` below 👍

Reply:
Actually remembering the reason I did this now, it was so we could pass `query: prefilledToken?.symbol` below without having coalescing empty string there: https://github.com/brave/brave-ios/blob/wallet/select-token-send-filter/Sources/BraveWallet/Crypto/Stores/SendTokenStore.swift#L135